web/src/pages/Setting/Model/SettingGeminiModel.js

```
@@ -10,6 +10,7 @@
   verifyJSON,
 } from '../../../helpers';
 import { useTranslation } from 'react-i18next';
+import Text from '@douyinfe/semi-ui/lib/es/typography/text.js';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Text'. Consider to delete it or rename it.

'Text' is defined here:


(lint/suspicious/noRedeclare)